examples/hosted-app-action-templates/javascript/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "devDependencies": {
+    "@contentful/node-apps-toolkit": "2.2.0",
```

Comment:
We don't use the dependency here. It's only used as type import for the typescript-one

Reply:
It's only used for types :)

Reply:
But it can be used to import methods 🤔   So users would need to move it over.

Reply:
I meant to comment on TS file. but which methods do you refer to?

Reply:
That's why we have hosted app actions in the first place 😛 We don't need any of that stuff anymore. Nor getting app tokens